blakserv/synched.c

```diff
@@ -577,6 +577,7 @@ void SynchedSendMenuChoice(session_node *s)
 void SynchedGotoGame(session_node *s,int last_download_time)
 {
    int num_new_files;
+   int move_interval = DEFAULT_MOVE_INTERVAL; 

```

Comment:
This default value is never used, so remove it.  Then declare the variable at the first point of use instead.

Reply:
Are you suggesting I just do away with the `DEFAULT_MOVE_INTERVAL` constant entirely?

```
AddByteToPacket(AP_GAME);

int move_interval = ConfigInt(UPDATE_MOVE_INTERVAL);
if (move_interval <= 0)
{
  AddIntToPacket(move_interval);
}

SendPacket(s->session_id);
```

I'm thinking yeah...

Reply:
If I did this, I'd need to update the client's packet handling of this. But now I'm doubting my thinking and wondering if always having the server send the move interval on game entry is the way to do this.